### ISSUE TITLE
Message stuck as sending on the web client

### ISSUE THREAD

**Issue by goxime** [klinker-apps/messenger-issues#920]
- [x] I have [searched](https://github.com/klinker-apps/messenger-issues/issues), by keyword, for my problem to verify that there aren't any open or closed issues around this behavior.
- [x] I have [searched the help page](https://messenger.klinkerapps.com/help) to verify that it does not have any topics relevant to this issue.
- [x] I have verified that this is something that is reproducible.

**Device with issue:** MacBook Air on MacOS 10.14, running Firefox 69.0 (with a new profile)
**Primary device:** Xiaomi Mi A2 (with microg)
**App version:** 4.11.9.2599

**Describe the bug:**
Messages submitted through the web client are stuck as "Sending..." and don't get sent by the device. Incoming messages are populated though.

I don't expect help with this since microg is an unsupported setup but others have reported Pulse SMS as working with it.

**Steps to reproduce:**

1. Install Pulse SMS on a device with Google Play Services
2. Buy the lifetime subscription
3. Get a new phone and install a custom rom with microg
4. Install Pulse SMS
5. Restore Pulse SMS data from your old phone using an oandbackup backup (data only)
6. Open My Account then "Resync Contacts", "Resync Messages", and "Refresh Account"
7. Log into the web interface and try to send a message
8. The message is stuck as "Sending..." and never gets sent or acknowledged by the primary device

**Expected behavior:**
The phone should receive the command to send the message and send it.

**Screenshots:**
![screenshot-20190918-221445](https://user-images.githubusercontent.com/55490198/65215259-cac37200-da61-11e9-8b22-0d2689ab73a0.png)

**Additional information:**
The primary device has microg and data was restored from an oandbackup from another phone.

**Comment by goxime** [comment]
It's now working.

I think this was caused by Pulse SMS not being registered for Google Cloud Messaging in microg. I noticed that Pulse SMS wasn't in the list of apps for that.

This is what I did to fix it:
1. Open microG Settings
2. Click Google Cloud Messaging
3. Click the kebab menu icon in the top right (three vertical dots)
4. Select Advanced from the menu
5. Uncheck Confirm new apps
6. Relaunch Pulse SMS
7. Pulse SMS is now shown in the list of registered apps for Google Cloud Messaging
8. Then re-enable Confirm new apps
